scripts/data_files/driver_jsons/driver_opaque_schema.json

```diff
@@ -0,0 +1,78 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "_comment": {
+      "type": "string"
+    },
+    "prefix": {
+      "type": "string",
+      "pattern": "^[A-Z_a-z][0-9A-Z_a-z]*$"
+    },
+    "type": {
+      "type": "string",
+      "const": ["opaque"]
+    },
+    "location": {
+      "type": ["integer","string"],
+      "pattern": "^(0x|0X)?[a-fA-F0-9]+$"
+    },
+    "mbedtls/h_depend_on": {
+      "type": "string"
+    },
+    "headers": {
+      "type": "array",
+      "items": {
+          "type": "string"
+        },
+        "minItems": 1,
+        "uniqueItems": true
+    },
+    "capabilities": {
+      "type": "array",
+      "items": [
+        {
+          "type": "object",
+          "properties": {
+            "_comment": {
+              "type": "string"
+            },
+            "mbedtls/c_depend_on": {
+              "type": "string"
+            },
+            "entry_points": {
+              "type": "array",
+              "items": {
+                  "type": "string",
+                  "enum": ["import_key", "export_key", "export_public_key",
+                          "copy_key", "get_builtin_key"]
+                },
+                "minItems": 1,
+                "uniqueItems": true
```

Comment:
I'm not sure it makes sense to forbid duplicates. In general, this list can contain names of entry point families as well as names of individual entry points, so uniqueness of entry points can't be determined syntactically anyway.

Goes for both types of drivers.